docs/approval-workflow.md

```diff
@@ -24,3 +26,118 @@ In order to activate the approval workflow the admin should go through the follo
 !**Important** The current workflow can be deactivated only if ALL orders in the system are in new, completed or rejected status. In case there is at least one order, which is still 'in progress', the approval workflow can not be deactivated or changed.
 
 ![Approval workflow](media/screen-approva-workflow-page.png)
```

Comment:
this is an outdated image too (here we have: new columns and actions). I think it will be better to describe these actions (workflow image preview, workflow getting as a JSON file)